development.ini.sample

```diff
@@ -135,6 +138,11 @@ meetings.timezone = America/Denver
 meetings.default_duration = 7200
 meetings.icalendar.prodid = -//Mines ACM//web//EN
 
+# Custom Assets Configuration
+custom_assets.dir = /
```

Comment:
It's a VERY bad idea to set the root of the filesystem as the custom assets dir, I would not reccomend this in the sample development.ini.

Try visiting /etc/passwd

Reply:
Just use ``site.custom_assets`` for this name